### DISCUSSION TITLE
does anyone know why there is a linker error when using spdlog:?

### DISCUSSION THREAD

**Discussion by TomerKigel** [gabime/spdlog#2813] [Q&A]
![image](https://github.com/gabime/spdlog/assets/88850793/0543dc64-277c-49c7-9e1e-b472f310e68e)

added the following image with the linker error.

I have added the header files in the additional include directories for the project for all configurations.
I call the header spdlog/spdlog.h in the desired place.
Yet in some files, it runs ok while in other files it raises the error above... what might be the issue?
help

**Comment by tt4g** [comment]
`MapViewOfFileNuma2` is Windows a API (https://learn.microsoft.com/en-us/windows/win32/api/memoryapi/nf-memoryapi-mapviewoffilenuma2).
I think it is a bug in MSVC or your project configuration problem.

**Comment by ymmfty0** [comment]
Add #pragma comment (lib, "OneCore.lib")
